### DISCUSSION TITLE
Wizard Back inputs disappear

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#10791] [Help]
Hello, i use Wizard in a form and when i came back my inputs disappear : 
First step : 
![image](https://github.com/filamentphp/filament/assets/77194719/fd32ab8b-71bd-4fd1-8b45-bd9a3935f929)
Second and go Back : 
![image](https://github.com/filamentphp/filament/assets/77194719/a2832ce4-080f-41c4-a056-0589213581c6)
I dont understand why, can you help me please ?

**Comment by leandrocfe** [comment]
Share the code please..
Any console errors?

**Comment by warsemann** [comment]
No error in console , for test i ve in create Page :

`    protected function getSteps(): array
    {
        return [

                Wizard\Step::make('Poste')
                ->schema([
                    Section::make()->schema(
                        [
                        Grid::make(3)
                        ->schema([
                            TextInput::make('first_name')
                                ->label('Prénom')
                                ->required()
                                ->maxLength(255),
                            TextInput::make('last_name')
                                ->label('Nom')
                                ->required()
                                ->maxLength(255)
                                ->autocapitalize()
                        ])
                        ]
                    )
                ]),

                Wizard\Step::make('Coordonnées')
                ->schema([
                    Section::make()
                        ->schema([
                            TextInput::make('first_name2')
                                ->label('Prénom')
                                ->required()
                                ->maxLength(255),
                            TextInput::make('last_name2')
                                ->label('Nom')
                                ->required()
                                ->maxLength(255)
                                ->autocapitalize()
                        ])
                ])
        ];
    }`

**Comment by warsemann** [comment]
Composer update solve m'y problem 😏
